arches_lingo/src/arches_lingo/components/generic/ResourceInstanceRelationshipsEditor.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import { computed, toRefs } from "vue";
+import MultiSelect from "primevue/multiselect";
+import type { ResourceInstanceReference } from "@/arches_lingo/types";
+
+const props = defineProps<{
+    val?: string[];
+    options?: ResourceInstanceReference[];
+}>();
+
+const emit = defineEmits(["update"]);
+
+const propRefs = toRefs(props);
+
+const value = computed({
+    get() {
+        return propRefs.val.value;
+    },
+    set(value) {
+        emit("update", value);
+    },
+});
+</script>
+<template>
+    <div>
```

Comment:
I like it because I like having a single dom element as a root, but I think that's just a preference I have from another life where it was required.  I'll yank it.